source/openapi-admin-v3.yaml

```diff
@@ -6811,22 +6946,22 @@ components:
           type: string
           description: The name of the App's Atlas Project.
         orgName:
-            type: string
-            description: The name of the App's Atlas Organization.
+          type: string
+          description: The name of the App's Atlas Organization.
         readPreference:
-            type: string
-            enum:
-              - primary
-              - primaryPreferred
-              - secondary
-              - secondaryPreferred
-              - nearest
+          type: string
+          enum:
+            - primary
+            - primaryPreferred
+            - secondary
+            - secondaryPreferred
+            - nearest
         wireProtocolEnabled:
-              type: boolean
-              description: |-
-                If true, clients may [connect to the app over the
-                MongoDB Wire
-                Protocol](https://www.mongodb.com/docs/atlas/app-services/mongodb/wire-protocol/#connect-over-the-wire-protocol).
+          type: boolean
+          description: |-
+            If true, clients may [connect to the app over the
+            MongoDB Wire
+            Protocol](https://www.mongodb.com/docs/atlas/app-services/mongodb/wire-protocol/#connect-over-the-wire-protocol).
     FlexibleSync:
```

Comment:
I notice we don't have required/optional on these fields. I'm missing `collection_queryable_fields_names` and `is_recovery_mode_disabled`. 

I also don't see anything about data ingest, and realize I don't know if you configure data ingest, how it shows up in the config object. I looked around and we don't seem to have anything documented for that. But I assume it shows up in the config somehow. 

Should we be more explicit about which of these fields may be absent? For a lot of our other config options, if there are no values, we just get an empty array - for example, my pull had empty arrays for triggers, values, log_forwarders, etc. So I might assume that `collection_queryable_fields_names` would show up as an empty array, and I'm not sure why `is_recovery_mode_disabled` is omitted.

Reply:
Added a bunch of required annotations. I think we'll want to do a separate ticket to look at ingest / sync config stuff